Crystal_FOX-FE_SMD.kicad_mod

```diff
@@ -1,23 +1,30 @@
-(module Crystal_FOX-FE_SMD (layer F.Cu)
-  (at 0 0)
+(module Crystal_FOX-FE_SMD (layer F.Cu) (tedit 0)
   (descr "Crystal, Quarz, FE, SMD, Vendor Fox,")
   (tags "Crystal Quarz FE SMD Vendor Fox")
   (attr smd)
   (fp_text reference REF** (at 0 -3.81) (layer F.SilkS)
     (effects (font (size 1 1) (thickness 0.15)))
   )
-  (fp_text value Crystal_FOX-FE_SMD (at 1.27 5.08) (layer F.Fab)
+  (fp_text value Crystal_FOX-FE_SMD (at 0 3.8) (layer F.Fab)
     (effects (font (size 1 1) (thickness 0.15)))
   )
+  (fp_line (start -4.7 3) (end -4.7 -3) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 4.7 3) (end -4.7 3) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 4.7 -3) (end 4.7 3) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -4.7 -3) (end 4.7 -3) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -3.7 2.5) (end -3.7 -2.5) (layer F.Fab) (width 0.05))
+  (fp_line (start 3.7 2.5) (end -3.7 2.5) (layer F.Fab) (width 0.05))
+  (fp_line (start 3.7 -2.5) (end 3.7 2.5) (layer F.Fab) (width 0.05))
+  (fp_line (start -3.7 -2.5) (end 3.7 -2.5) (layer F.Fab) (width 0.05))
   (fp_circle (center 0 0) (end 0.8001 0) (layer F.Adhes) (width 0.381))
   (fp_circle (center 0 0) (end 0.50038 0) (layer F.Adhes) (width 0.381))
   (fp_circle (center 0 0) (end 0.14986 -0.0508) (layer F.Adhes) (width 0.381))
-  (fp_line (start -3.74904 2.49936) (end -3.74904 1.69926) (layer F.SilkS) (width 0.15))
-  (fp_line (start -3.74904 -2.49936) (end -3.74904 -1.69926) (layer F.SilkS) (width 0.15))
-  (fp_line (start 3.74904 -2.49936) (end 3.74904 -1.69926) (layer F.SilkS) (width 0.15))
-  (fp_line (start 3.74904 2.49936) (end 3.74904 1.69926) (layer F.SilkS) (width 0.15))
-  (fp_line (start -3.74904 2.49936) (end 3.74904 2.49936) (layer F.SilkS) (width 0.15))
-  (fp_line (start 3.74904 -2.49936) (end -3.74904 -2.49936) (layer F.SilkS) (width 0.15))
+  (fp_line (start -3.8 2.6) (end -3.8 1.7) (layer F.SilkS) (width 0.15))
+  (fp_line (start -3.8 -2.6) (end -3.8 -1.69926) (layer F.SilkS) (width 0.15))
```

Comment:
you should move the end point a little bit, to be the same as all the other ones